pkg/server/api/svid/v1/service_test.go

```diff
@@ -1734,14 +1735,14 @@ func TestServiceBatchNewX509SVID(t *testing.T) {
 
 				require.Equal(t, entry.DnsNames, svid.DNSNames)
 
-				expectedSubject := &pkix.Name{Country: []string{"US"}, Organization: []string{"SPIRE"}}
```

Comment:
I think the code is covered by mint... but maybe you can add a test case here where CA has a subject, and we can see than our SVID is inheriting it?

Reply:
No longer relevant since we've removed CA subject inheritance.